migtests/tests/mysql/basic-live-test/validate

```diff
@@ -1,10 +1,21 @@
 #!/usr/bin/env python3
 
+import argparse
 import os
 import yb
 import collections
 
 def main():
+	parser = argparse.ArgumentParser(description='Process an argument.')
```

Comment:
**If possible** can we have this black as a function in yb.py? It will reduce the code duplication.

Reply:
Hmm, not in yb.py and oracle.py but another separate library (common.py) that can have common functions among validate files. Can refactor this later.